server.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
```

Comment:
`'use strict'` is good but it should be in all of your `.js` files if you're opting to use it